### PR TITLE
Add non-null to fields

### PR DESCRIPTION
**Pull request by xuorig**
forgot about non nulls! Pretty sure all these fields are non-null, havent verified all of them but worst case well fix when we see errors. @cjoudrey

### CODE REVIEW

File: app/models/graph/types/person.rb [hunk 1]
```diff
@@ -8,26 +8,26 @@ module Types
...
 
-      field :birthYear, types.String,
+      field :birthYear, !types.String,
         "The birth year of the person, using the in-universe standard of BBY or ABY"\
         " - Before the Battle of Yavin or After the Battle of Yavin. The Battle of Yavin"\
         " is a battle that occurs at the end of Star Wars episode IV: A New Hope.",
         property: :birth_year
 
-      field :eyeColor, types.String,
+      field :eyeColor, !types.String,
```
Comment (cjoudrey) [inline app/models/graph/types/person.rb]:
```
> Person.where(eye_color: nil).map(&:name)
  Person Load (0.4ms)  SELECT "people".* FROM "people" WHERE "people"."eye_color" IS NULL
=> ["Wat Tambor", "Ratts Tyerell", "Captain Phasma"]
```

File: app/models/graph/types/person.rb [hunk 2]
```diff
@@ -22,16 +22,16 @@ module Types
...
-      field :height, types.Int, "The height of the person in centimeters."
-      field :homeworld, Planet, "A planet that this person was born on or inhabits."
-      field :mass, types.Int, "The mass of the person in kilograms."
-      field :name, types.String, "The name of this person."
-      field :skinColor, types.String, "The skin color of this person.", property: :skin_color
+      field :height, !types.Int, "The height of the person in centimeters."
+      field :homeworld, !Planet, "A planet that this person was born on or inhabits."
+      field :mass, !types.Int, "The mass of the person in kilograms."
+      field :name, !types.String, "The name of this person."
+      field :skinColor, !types.String, "The skin color of this person.", property: :skin_color
```
Comment (cjoudrey) [inline app/models/graph/types/person.rb]:
```
> Person.where(skin_color: nil).map(&:name)
  Person Load (0.2ms)  SELECT "people".* FROM "people" WHERE "people"."skin_color" IS NULL
=> ["Chewbacca", "Captain Phasma"]
```

File: app/models/graph/types/person.rb [hunk 3]
```diff
@@ -22,16 +22,16 @@ module Types
...
       field :hairColor, types.String,
         "The hair color of this person. Will be \"unknown\" if not known or \"n/a\" if the person does not have hair.",
         property: :hair_color
 
-      field :height, types.Int, "The height of the person in centimeters."
-      field :homeworld, Planet, "A planet that this person was born on or inhabits."
-      field :mass, types.Int, "The mass of the person in kilograms."
-      field :name, types.String, "The name of this person."
-      field :skinColor, types.String, "The skin color of this person.", property: :skin_color
+      field :height, !types.Int, "The height of the person in centimeters."
```
Comment (cjoudrey) [inline app/models/graph/types/person.rb]:
```
> Person.where(height: nil).map(&:name)
  Person Load (0.2ms)  SELECT "people".* FROM "people" WHERE "people"."height" IS NULL
=> ["Arvel Crynyd", "BB8", "Poe Dameron", "Rey", "Captain Phasma", "Finn"]
```

File: app/models/graph/types/person.rb [hunk 4]
```diff
@@ -22,16 +22,16 @@ module Types
...
+      field :homeworld, !Planet, "A planet that this person was born on or inhabits."
+      field :mass, !types.Int, "The mass of the person in kilograms."
```
Comment (cjoudrey) [inline app/models/graph/types/person.rb]: Same here :(

File: app/models/graph/types/person.rb [hunk 5]
```diff
@@ -22,16 +22,16 @@ module Types
...
         "The hair color of this person. Will be \"unknown\" if not known or \"n/a\" if the person does not have hair.",
         property: :hair_color
 
-      field :height, types.Int, "The height of the person in centimeters."
-      field :homeworld, Planet, "A planet that this person was born on or inhabits."
-      field :mass, types.Int, "The mass of the person in kilograms."
-      field :name, types.String, "The name of this person."
-      field :skinColor, types.String, "The skin color of this person.", property: :skin_color
+      field :height, !types.Int, "The height of the person in centimeters."
+      field :homeworld, !Planet, "A planet that this person was born on or inhabits."
```
Comment (cjoudrey) [inline app/models/graph/types/person.rb]: Here too :(

File: app/models/graph/types/planet.rb [hunk 6]
```diff
@@ -11,26 +11,26 @@ module Types
       connection :films, Graph::Types::Film.connection_type
       connection :residents, Graph::Types::Person.connection_type
 
-      field :name, types.String, "The name of this planet."
-      field :diameter, types.Int, "The diameter of this planet in kilometers."
+      field :name, !types.String, "The name of this planet."
+      field :diameter, !types.Int, "The diameter of this planet in kilometers."
```
Comment (cjoudrey) [inline app/models/graph/types/planet.rb]:
```
> Planet.where(diameter: nil).count
   (0.2ms)  SELECT COUNT(*) FROM "planets" WHERE "planets"."diameter" IS NULL
=> 17
```

😭

File: app/models/graph/types/planet.rb [hunk 7]
```diff
@@ -11,26 +11,26 @@ module Types
...
 
-      field :rotationPeriod, types.Int,
+      field :rotationPeriod, !types.Int,
```
Comment (cjoudrey) [inline app/models/graph/types/planet.rb]:
```
> Planet.where(rotation_period: nil).count
   (0.2ms)  SELECT COUNT(*) FROM "planets" WHERE "planets"."rotation_period" IS NULL
=> 13
```

:sob:

File: app/models/graph/types/planet.rb [hunk 8]
```diff
@@ -11,26 +11,26 @@ module Types
...
         "The number of standard hours it takes for this planet to complete a single rotation on its axis.",
         property: :rotation_period
 
-      field :orbitalPeriod, types.Int,
+      field :orbitalPeriod, !types.Int,
```
Comment (cjoudrey) [inline app/models/graph/types/planet.rb]:
```
> Planet.where(orbital_period: nil).count
   (0.2ms)  SELECT COUNT(*) FROM "planets" WHERE "planets"."orbital_period" IS NULL
=> 13
```
surprise! :sob:

File: app/models/graph/types/planet.rb [hunk 9]
```diff
@@ -11,26 +11,26 @@ module Types
...
         "The number of standard days it takes for this planet to complete a single orbit of its local star.",
         property: :orbital_period
 
-      field :gravity, types.String,
+      field :gravity, !types.String,
```
Comment (cjoudrey) [inline app/models/graph/types/planet.rb]: This one too.

File: app/models/graph/types/planet.rb [hunk 10]
```diff
@@ -11,26 +11,26 @@ module Types
...
 
-      field :population, types.Float, "The average population of sentient beings inhabiting this planet."
-      field :climate, types.String, "The climate of this planet."
-      field :terrain, types.String, "The terrain of this planet."
+      field :population, !types.Float, "The average population of sentient beings inhabiting this planet."
+      field :climate, !types.String, "The climate of this planet."
+      field :terrain, !types.String, "The terrain of this planet."
 
-      field :surfaceWater, types.Float,
+      field :surfaceWater, !types.Float,
```
Comment (cjoudrey) [inline app/models/graph/types/planet.rb]: `population`, `climate`, `terrain`, `surfaceWater`. All of these too. :sob:

File: app/models/graph/types/species.rb [hunk 11]
```diff
@@ -11,11 +11,11 @@ module Types
       connection :people, Graph::Types::Person.connection_type
       connection :films, Graph::Types::Film.connection_type
 
-      field :name, types.String, "The name of this species."
-      field :classification, types.String, "The classification of this species, such as \"mammal\" or \"reptile\"."
-      field :designation, types.String, "The designation of this species, such as \"sentient\"."
+      field :name, !types.String, "The name of this species."
+      field :classification, !types.String, "The classification of this species, such as \"mammal\" or \"reptile\"."
```
Comment (cjoudrey) [inline app/models/graph/types/species.rb]:
```
> Species.where(classification: nil).count
   (1.3ms)  SELECT COUNT(*) FROM "species" WHERE "species"."classification" IS NULL
=> 6
```

File: app/models/graph/types/species.rb [hunk 12]
```diff
@@ -41,16 +41,16 @@ module Types
...
         resolve ->(species, _, _) do
           return unless colors = species.skin_colors
           colors.split(", ")
         end
       end
 
-      field :language, types.String, "The language commonly spoken by this species."
-      field :homeworld, Graph::Types::Planet, "A planet that this species originates from type."
+      field :language, !types.String, "The language commonly spoken by this species."
+      field :homeworld, !Graph::Types::Planet, "A planet that this species originates from type."
```
Comment (cjoudrey) [inline app/models/graph/types/species.rb]:
```
  (1.9ms)  SELECT COUNT(*) FROM "species" WHERE "species"."homeworld_id" IS NULL
=> 2
```

File: app/models/graph/types/species.rb [hunk 13]
```diff
@@ -41,16 +41,16 @@ module Types
...
         description "Common skin colors for this species, null if this species does not typically have skin."
         resolve ->(species, _, _) do
           return unless colors = species.skin_colors
           colors.split(", ")
         end
       end
 
-      field :language, types.String, "The language commonly spoken by this species."
-      field :homeworld, Graph::Types::Planet, "A planet that this species originates from type."
+      field :language, !types.String, "The language commonly spoken by this species."
```
Comment (cjoudrey) [inline app/models/graph/types/species.rb]:
```
   (0.1ms)  SELECT COUNT(*) FROM "species" WHERE "species"."language" IS NULL
=> 1
```

**Comment by xuorig** [comment]
merging, only name is non null except for film

**Pull request merged**